target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,9 @@ case "$(board_name)" in
 pc-engines-apu1|pc-engines-apu2|pc-engines-apu3)
 	ucidef_set_interfaces_lan_wan "eth1 eth2" "eth0"
 	;;
+sophos-sg-105|sophos-xg-105)
+	ucidef_set_interfaces_lan_wan "eth0 eth2 eth3" "eth1"
```

Comment:
Curious order, is that normal?

Reply:
Yes, that corresponds to marking on the enclosure: LAN WAN DMZ OPT.